sourcetracker/_cli/gibbs.py

```diff
@@ -144,7 +167,16 @@ def gibbs(table_fp: Table,
           sink_column_value: str,
           source_category_column: str,
           diagnostics: bool,
-          limit: float):
+          limit: float,
+          stacked_bar: bool,
+          heatmap: bool,
+          paired_heatmap: bool,
+          title: str,
+          heatmap_color: str,
+          unknowns: bool,
```

Comment:
```suggestion
          keep_unknowns: bool,
```
I think we should call this keep_unknowns because its more interpretable for users than unknown.